engine.py

```diff
@@ -681,60 +681,6 @@ def _init_pyside(self):
             self.logger.debug("PySide2 detected - the existing version will be used.")
             return
 
-        # Then see if pyside is present
```

Comment:
We should update line `678` since we no longer try to import PySide anymore.

Also, if neither PySide6 or PySide2 are found, should we raise some kind of error? Else this will kind of silently fail, only debug messages will be logged to say that PySide6/2 were not detected